packages/contracts/.local.env

```diff
@@ -0,0 +1,6 @@
+DEPLOY_MNEMONIC='social spell miracle box cream brisk dash edit badge citizen tail modify'
```

Comment:
+1 for high quality mnemonic, we are definitely building a social spell miracle box

Reply:
OH NO! Everybody knows my ganache key!

but for real... this should have been removed by the .gitignore. I'll manually remove it.